blockbase/create-child.js

```diff
@@ -0,0 +1,151 @@
+const child_theme_json = `{
+	"settings": {
+		"custom": {}
+	},
+	"styles": {
+		"blocks": {}
+	}	
+}`;
+
+const child_package_json = `{
+  "name": "{newtheme}",
+  "version": "0.0.1",
+  "description": "",
+  "main": "style.css",
+  "dependencies": {},
+  "devDependencies": {
+    "@wordpress/base-styles": "^3.4.0",
+    "chokidar-cli": "^2.1.0",
+    "node-sass": "^5.0.0",
+    "node-sass-package-importer": "^5.3.2"
+  },
+  "scripts": {
+    "start": "chokidar \\"**/*.scss\\" child-theme.json ../blockbase/theme.json -c \\"npm run build\\" --initial",
+    "build": "npm run build:theme && npm run build:scss",
+    "build:theme": "node ../blockbase/build.js {newtheme}",
+    "build:scss": "node-sass --importer node_modules/node-sass-package-importer/dist/cli.js sass/theme.scss assets/theme.css --output-style expanded --indent-type tab --indent-width 1 --source-map true"
+  },
+  "author": "",
+  "license": "GPLv2"
+}`;
+
+const style_css = `/*
+Theme Name: {newtheme}
+Theme URI: 
+Author: 
+Author URI: 
+Description: 
+Requires at least: 5.7
+Tested up to: 5.7
+Requires PHP: 5.7
+Version: 0.0.1
+License: GNU General Public License v2 or later
+License URI: 
+Template: blockbase
+Text Domain: {newtheme}
+Tags: 
+
+*/`;
+
+const theme_scss = `	
+// Custom CSS should be added here.  It will be compiled to /assets/theme.css.
+`;
+
+const functions_php = `
+<?php
+
+/**
+ * Add Editor Styles
+ */
+function newtheme_editor_styles() {
+	// Enqueue editor styles.
+	add_editor_style(
+		array(
+			'/assets/theme.css',
+		)
+	);
+}
+add_action( 'after_setup_theme', 'newtheme_editor_styles' );
+
+/**
+ *
+ * Enqueue scripts and styles.
+ */
+function newtheme_scripts() {
+	wp_enqueue_style( 'newtheme-styles', get_stylesheet_directory_uri() . '/assets/theme.css', array('blockbase-ponyfill'), wp_get_theme()->get( 'Version' ) );
+}
+add_action( 'wp_enqueue_scripts', 'newtheme_scripts' );
+
+`;
+
+const block_templates_index_html = `
```

Comment:
Can we copy these from Blockbase so that if they change we don't have to update here?

Reply:
...maybe.
The files for a child theme aren't the same files blockbase uses (or a derivative theme would use).
If we wanted to copy files instead of have them in this script then we would still need to copy files that aren't the ones blockbase uses, and would still have to change them independently... so it's just moving the contents from one place to another.  Which may be better! But I don't think it addresses "if they change we don't have to update the generation".

Reply:
I just meant the block templates, not the other files. Are the block template different?

Reply:
Yea, they are more descriptive than functional.
`<span>Put yer templates here.  Here's an example of where to put 'em and how to use template parts</span>`